cmd/utils.go

```diff
@@ -462,3 +462,27 @@ func getPrometheusToken(hostConfig *aliasConfigV10) (string, error) {
 	}
 	return token, nil
 }
+
+func removeOverlappingPrefixes(prefixList []string) (cleanList []string) {
```

Comment:
I see your point, didn't really think about someone using it in that way but since it's possible..

Reply:
@zveinn You learn to CYA - if there is a way to abuse, it *will* be taken :)

Reply:
I took a step back and kind of re-though how this could be implemented. Ended up completely re-writing all of it into a filter channel that intercepts and filters duplicate URLs as they are being processed.